server/middlewares/helperFunction.js

```diff
@@ -1,16 +1,20 @@
 import orderData from '../models/order';
 
 /**
- * @desc function is created to prevent de-duplication of code in the get one and update order
+ * @desc helper function that returns error if resource is not found
  * @param {object} req - the request object
  * @param {object} res - the response object
  * @param {*} next - the callback function
  * @returns {object} - status code and error message
  */
 function Helper(req, res, next) {
-  const index = orderData.findIndex(order => order.id === Number(req.params.id));
+  const index = orderData.findIndex(
+    order => order.id === Number(req.params.id)
+  );
   const error = 'order not found';
-  if (index === -1) return res.status(404).send({ status: 'error', error });
+  if (index === -1) {
+    return res.status(404).send({ status: 'error', error });
+  }
   return next();
 }
 export default Helper;
```

Comment:
The name of this export has to be the same as the name of the file i.e `helperFunction.js`. I also don't think helperFunction is a descriptive enough name. What does this helper function do?

Reply:
The function helps me prevent code duplication and it basically checks if a resource exists when the GET and PUT endpoints are accessed